### PR TITLE
Create a demo container

### PR DESCRIPTION
**Pull request by apmasell**
* Create a Shesmu demo instance
 This is demo data and olives for users who want to export Shesmu.
* Install fonts in Docker so that Shesmu can render metro diagrams

### CODE REVIEW

File: demo/README.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+have to a lot of complicated grouping and processing to handle mismatches
+between barcodes and bases masks. Additionally, MISO only provides some
+information about Illumina's flowcell architecture, so Shesmu is responsible
+for determining which lanes are loaded by a single port.
+
+If there are no reads for a particular IUS, we send a JIRA ticket to the lab to
+determine the source of the problem. This olive is in
+`ticket-missingreads.shesmu`.
+
+Once a workflow runs, its output is capture by our file provenance system and
```
Comment (morgantaschuk) [inline demo/README.md:47]: sp - captured

File: demo/README.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+customer configuration that our systems need. We also need to make decisions
+based on the lab work and `kit_info.jsonconfig` holds information about
+different processing kits out lab uses and provides information for using them.
+
+For demonstration, we have included `niassa-bwamem.shesmu` which does alignment
+on the FASTQs generated by BCL2FASTQ and `niassa-bmpp.shesmu`, which collects
+related BAMs for merging multiple BAMs from the same donor and performing
+co-cleaning on a per-project basis.
+
+We have no provided the workflows for any of these; merely the minimal
```
Comment (morgantaschuk) [inline demo/README.md:62]: sp - not

File: demo/README.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+information Shesmu needs as a `.fakeaction` file. The full workflows are
+available on [OICR GSI](https://github.com/oicr-gsi), but we have elided our
```
Comment (morgantaschuk) [inline demo/README.md:64]: is 'elided' the right word here?

File: demo/README.md [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+olive will be displayed and numbers, indicating the number of records
+associated with each clause in an olive.
+
+There is an olive simulator available from the _Tools_ menu. On the _Olives_
+dashboard, it is possible to preload the simulator with an existing olive
+script using the _Edit in Simulator_ link.
+
+On the _Actions_ page, a similar view to the actions on the _Olives_ page is
+shown, but the output of all olives are mixed together. It is possible to
+filter the displayed actions using the _Add Filter_ button. Since the demo, all
```
Comment (morgantaschuk) [inline demo/README.md:89]: Fix sentence "Since the demo"

File: demo/README.md [hunk 5]
```diff
@@ -0,0 +1,101 @@
...
+actions will be in a _ZOMBIE_ state, but multiple states would be present in a
+server with real actions.
+
+The actions display, on both the _Olives_ and _Actions_ pages, a variety of
+filters can limit the actions shown. The current filter is saved in the URL, so
```
Comment (morgantaschuk) [inline demo/README.md:94]: filters that can

File: demo/demo.fakeactiondefs [hunk 6]
```diff
@@ -0,0 +1,278 @@
+[
+  {
+    "description": "Runs SeqWare/Niassa workflow 17318036 with settings in /home/debian/shesmu/production/gsi.niassa. Considered equivalent to workflows: 2280235, 2967897, 11971238, 13262061, 15033907, 15078370, 15078370, 16543238, 16563239, 16733247",
```
Comment (morgantaschuk) [inline demo/demo.fakeactiondefs:3]: It's trivial but can you wipe "SeqWare" out of the whole demo?

File: demo/demo.fakeactiondefs [hunk 7]
```diff
@@ -0,0 +1,278 @@
...
+      },
+      {
+        "name": "type",
+        "required": false,
+        "type": "s"
+      }
+    ]
+  },
+  {
+    "description": "Closes any JIRA tickets with a matching summary. Defined in /home/debian/shesmu/production/genomics.jira.",
```
Comment (morgantaschuk) [inline demo/demo.fakeactiondefs:253]: Is this path correct?

File: demo/niassa-bwaMem.shesmu [hunk 8]
```diff
@@ -0,0 +1,342 @@
...
+Function get_bwamem_params(string reference)
+  Switch reference
+    When "hg19" Then
+      `{
+        bwaRef = "$HG19_BWA_INDEX_ROOT/hg19_random.fa",
+        modules = "samtools/1.9 bwa/0.7.12 hg19-bwa-index/0.7.12"
+      }`
+    When "hg38" Then
+    `{
+        bwaRef = "$HG38_BWA_INDEX_ROOT//hg38_random.fa",
```
Comment (morgantaschuk) [inline demo/niassa-bwaMem.shesmu:14]: Double //

File: demo/demo.fakeactiondefs (hunk 7 above)
Comment (apmasell) [reply demo/demo.fakeactiondefs:253]: Intentionally no.

**Pull request merged**